Source/hydro/Castro_hydro.cpp

```diff
@@ -23,8 +23,7 @@ Castro::construct_hydro_source(Real time, Real dt)
 
     sources_for_hydro.setVal(0.0);
 
-    for (int n = 0; n < num_src; ++n)
-	MultiFab::Add(sources_for_hydro, *old_sources[n], 0, 0, NUM_STATE, NUM_GROW);
+    MultiFab::Add(sources_for_hydro, old_sources, 0, 0, NUM_STATE, NUM_GROW);
```

Comment:
since `old_sources` is already the sum, do we still need a separate `sources_for_hydro`?

Reply:
Yes, because we need a separate container to hold this so that we can add dSdt. Note also that I plan to make the sources data StateData (and to have zero ghost cells), so we'll need sources_for_hydro around so we can FIllPatch into a non-zero number of ghost zones.